assignments/objects.js

```diff
@@ -19,27 +19,81 @@ const example = {
 
 // Write your intern objects here:
 
+const intern = [
```

Comment:
Can you try making separate objects instead of an array? Think about what you would have to change.